paasta_tools/cli/cmds/logs.py

```diff
@@ -698,7 +686,7 @@ def filter_and_aggregate_scribe_logs(self, scribe_reader_ctx, scribe_env, stream
                     if filter_fn:
                         if filter_fn(line, levels, service, components, clusters,
                                      start_time=start_time, end_time=end_time):
-                            aggregated_logs.append(line)
+                            print_log(line, levels, raw_mode)
```

Comment:
but if you print right away without aggregating, will the logs be in the right order?